### ISSUE TITLE
Robotics Sphero Track

### ISSUE THREAD

**Issue by c-gale** [c-gale/Robotics-1#2]
# Robotics Lab book
## Project Description:
Sphero Track, 5/1/24, Engineered by Cole Gale.
### Goal
The Goal of the assignment is to make a Sphero go around set points on a track represented by cones. The Sphero has to go around the cones, so go to the Left of the cone to go Right and vice versa.
## Required Parts:
Name | Quantity | Component
-- | -- | --
S1 | 1 | Sphero Spark+

Code:
```js
async function startProgram() {
	setMainLed({ r: 163, g: 124, b: 0 });
	await roll(0, 135, 3.1);
	await delay(1);
	setMainLed({ r: 255, g: 76, b: 115 });
	await roll(97, 135, 3.9);
	await delay(1);
	setMainLed({ r: 193, g: 255, b: 202 });
	await roll(4, 135, 3.3);
	await delay(1);
	setMainLed({ r: 52, g: 73, b: 255 });
	await roll(265, 135, 4.2);
	await delay(1);
	setMainLed({ r: 100, g: 0, b: 255 });
	await roll(7, 135, 3.5);
	await delay(1);
	setMainLed({ r: 255, g: 252, b: 24 });
	await roll(276, 135, 2.8);
	await delay(1);
	setMainLed({ r: 15, g: 255, b: 0 });
	await roll(181, 135, 4.5);
	await delay(1);
	setMainLed({ r: 255, g: 85, b: 0 });
	await roll(87, 135, 2.7);
	await delay(1);
	setMainLed({ r: 255, g: 19, b: 0 });
	await roll(176, 135, 4.3);
}
```

### Issue 1:
The Sphero’s movement is unpredictable at times.
### Solution 1:
Tweak until better results

### Issue 2:
Slight differences in the start position and aim causes the Sphero to create separate results
### Solution 2:
Using a piece of paper to mark where the Sphero should be, in practice, increased the success a lot more.

### Issue 3:
The marker (piece of paper) moves over time
### Solution 3:
Tape the marker down to ensure it stays in the same place no matter what.

Conclusion:
In conclusion If I ever do another project that requires trial and error or a lot of tweaking I will try to ensure the start conditions are as close per iteration.
